charts/minikube.md

```diff
@@ -0,0 +1,55 @@
+### Requirements
+- [minikube](https://github.com/kubernetes/minikube/#installation)
+  - [virtualbox](https://www.virtualbox.org/wiki/Downloads)
+- [helm](https://github.com/kubernetes/helm#install)
+
+The following assumes you have minikube (which requires virtualbox or other virtualization options)
+as well as helm installed (see links above). Once minikube and helm are installed, start minikube 
+with at least version 1.9 of kubernetes and perferably with 4GB of RAM (although the default value 
+of 2GB should work, we recommend 4GB or more)
+```shell
+minikube start --kubernetes-version=v1.9.0 --memory 4096
+```
+
+
+Once minikube has started, run the helm initialization. 
+#### NOTE that once you helm init, it may take a couple minutes for the Tiller services to fully deploy on your minikube
+```
+helm init
+```
+
+Clone the repo and go into the charts directory
+```shell
+git clone https://github.com/scality/Zenko.git
+cd Zenko/charts
+```
+
+Then once you have the repo cloned you can retrieve all dependencies:
+
+```shell
+$ helm repo add zenko-zookeeper https://scality.github.io/zenko-zookeeper/charts
+"zenko-zookeeper" has been added to your repositories
+
+$ helm dependency build zenko/
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "zenko-zookeeper" chart repository
+...Successfully got an update from the "stable" chart repository
+Update Complete. ⎈Happy Helming!⎈
+Saving 1 charts
+Downloading zenko-zookeeper from repo https://scality.github.io/zenko-zookeeper/charts
+Deleting outdated charts
+```
+
+With your dependencies built, you can run the following shell command to deploy the zenko stack with orbit enabled.
+```shell
+helm install --name zenko --set ingress.enabled=true  --set cloudserver-front.orbit.enabled=true -f single-node-values.yml zenko
```

Comment:
Why pass `--set` values when you provide a tailored YAML configuration anyway?

Reply:
I hadn't considered that. I'll set these as the defaults in the single node values.